src/components/specific/visa/visa-summary/visa-summary-people/VisaSummaryPeople.vue

```diff
@@ -5,69 +5,149 @@
+          people.isSelf &&
+          people.status != VALIDATION_STATUS.PENDING
+        "
+      >
         <BIMDataButton
-          color="silver"
           radius
           ghost
-          margin="0 10px 0 0"
-          @click="$emit('reset-visa')"
+          style="margin: 2px 0px 0px 0px"
+          @click="$emit('reset-val', people.id)"
         >
           <BIMDataIcon name="reset" size="xs" />
         </BIMDataButton>
       </template>
-      <span>
-        {{
-          $t(
-            `Visa.summary.people.${
-              people.status === STATUS.PENDING
-                ? "pending"
-                : people.status === STATUS.ACCEPT
-                ? "accept"
-                : "deny"
-            }`
-          )
-        }}
-      </span>
-      <BIMDataIcon
-        :name="
-          people.status === STATUS.PENDING
-            ? 'sandglass'
-            : people.status === STATUS.ACCEPT
-            ? 'visa'
-            : 'failed'
-        "
-        size="xs"
-      />
+      <template v-if="people.hasAccess">
+        <div :class="`visa-add-summary-people__right-side__${people.status}`">
+          <span>
+            {{
+              $t(
+                `Visa.summary.people.${
+                  people.hasCommented &&
+                  people.status === VALIDATION_STATUS.PENDING
+                    ? "commented"
+                    : people.status === VALIDATION_STATUS.PENDING
+                    ? "pending"
+                    : people.status === VALIDATION_STATUS.ACCEPT
+                    ? "accept"
+                    : "deny"
+                }`
+              )
+            }}
+          </span>
+          <BIMDataIcon
+            :name="
+              people.hasCommented && people.status === VALIDATION_STATUS.PENDING
+                ? 'comment'
+                : people.status === VALIDATION_STATUS.PENDING
+                ? 'sandglass'
+                : people.status === VALIDATION_STATUS.ACCEPT
+                ? 'visa'
+                : 'failed'
+            "
+            size="xs"
+            style="margin-top: 2px"
+          />
+        </div>
+      </template>
+      <template v-else>
+        <div
+          class="visa-add-summary-people__right-side__access-denied"
+          :style="isAuthor ? 'right: -8px;' : 'right: -12px'"
+          @mouseover="handleCurrentPerson(people.id)"
+          @mouseleave="handleCurrentPerson()"
+        >
+          <BIMDataIcon
+            name="warning"
+            customSize="40"
+            class="fill-warning"
+            style="padding: 10px; z-index: 3"
+          />
+          <div
+            v-if="isWarningHover && currentPeopleId === people.id"
+            class="visa-add-summary-people__right-side__access-denied__hover"
+            :style="isAuthor ? 'width: 345px; right: -36px' : 'width: 340px'"
+          >
+            <span>{{ $t("Visa.selectionValidator.warning") }}</span>
+          </div>
+        </div>
```

Comment:
I see that you use many inline styles here. You could rewrite this in a more concise way:

```html
<div
   class="visa-add-summary-people__right-side__access-denied"
   :class="{ isAuthor }"
   ...
>
   <BIMDataIcon
      class="fill-warning"
      name="warning"
      ...
   />
   <div
      v-if="isWarningHover && currentPeopleId === people.id"
      class="visa-add-summary-people__right-side__access-denied__hover"
   >
      <span>{{ $t("Visa.selectionValidator.warning") }}</span>
   </div>
</div>
```

With the following styles:

```scss
...
   .visa-add-summary-people__right-side__access-denied {
      ...
      right: -12px;
      
      .bimdata-icon {
         z-index: 3; // I'm not sure this is actually useful
         padding: 10px;
      }
      
      &__hover {
         width: 340px;
      }
      
      &.isAuthor {
          right: -8px;
         .visa-add-summary-people__right-side__access-denied__hover {
            right: -36px;
            width: 345px;
         }
      }
   }
...
```